app/templates/components/project-join-modal.hbs

```diff
@@ -0,0 +1,27 @@
+{{#if show}}
+  {{#modal-dialog clickOutsideToClose=true
+                  close="closeModal"
+                  containerClassNames="project-join-modal"
+                  targetAttachment="center"
+                  translucentOverlay=true}}
+    <div>
+      <h1>Join {{project.title}}</h1>
+      <div class="project-join-modal__icons">
+        <div class="project-join-modal__icons__avatar">
+          <img class="icon medium" src={{user.photoLargeUrl}} />
+        </div>
+        <div class="project-join-modal__icons__icon project-join-modal__icons__icon--join"></div>
+        <div class="project-join-modal__icons__avatar">
+          <img class="icon medium" src={{project.iconLargeUrl}} />
+        </div>
+      </div>
+      <div class="project-join-modal__skills">
+        <p>What can you help with?</p>
+        {{related-skills isClickable=true skills=skills}}
```

Comment:
This `isClickable` is not at all tested on a couple of the higher-level components.